CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+<a name="1.71.2"></a>
+## 1.71.2 (2024-04-17)
+
+
+#### Bug Fixes
+
+*   a couple retry logic error messages were incorrect (#676) ([6cb6549b](https://github.com/mozilla-services/autopush-rs/commit/6cb6549b087a3ff44262cd796e45ec50d231cd37))
+
+#### Features
+
+*   Ensure that status and labels are propagated for DB errors (#679) ([5a1f2099](https://github.com/mozilla-services/autopush-rs/commit/5a1f20997ad1705c6ded3ace02f6f1a1be5a70ef))
+*   emit pool timeouts/conditional failures as metrics (not sentry) (#671) ([b4f01af8](https://github.com/mozilla-services/autopush-rs/commit/b4f01af8f68b157a39ec3bf4bfa2778f283652c0))
+*   Switch to randomized UAID for health check (#670) ([06093787](https://github.com/mozilla-services/autopush-rs/commit/060937875b34695d8c2be39425c35a1b71e007aa))
+
+#### Test
+
+* **load:**  Document the process for calibrating users per worker  (#661) ([c89627fd](https://github.com/mozilla-services/autopush-rs/commit/c89627fdad2708613acd59f480d44913f074b6a5))
+
+#### Doc
+
+* **load:**  Document the process for calibrating worker count (#666) ([de752ec4](https://github.com/mozilla-services/autopush-rs/commit/de752ec4e118c7218c461d9c2488529fc71f8a04))
+
```

Comment:
```suggestion
**testing:** Update the broken and out of date autopush testing docs. (#674) ([d1fe465](https://github.com/mozilla-services/autopush-rs/commit/d1fe465118add06ba838fdaccd2d23c69addc24c))
```

Reply:
Hmm. these normally come out of running `clog`, which looks for specific prefixes. These are defined in the [CONTRIBUTING](https://github.com/mozilla-services/autopush-rs/blob/master/CONTRIBUTING.md#git-commit-guidelines) doc. We kinda screwed up by not enforcing the `test` prefix for the commit message on those commits. (Changing them in the github PR doesn't always ensure that the `git log` has the correct text.)

We should be better about that so we don't miss future changes.

Reply:
:+1: we generally don't edit the clog output at all (except for an occasional removal) -- this is a good reminder to ensure the commit log will reflect what's wanted in the CHANGELOG during the PR merge. Also don't forget to squash the commit if necessary -- a reminder to myself, because I often forget the squash, littering the commit log 🤦‍♂️

Reply:
@pjenvey @jrconlin Great, thanks for the context, I'll be aware of that going forward.
Question though: since #674  has the branch name `docs/SYNC-4001_update_outdated_test_docs`, shouldn't that mean `clog` should have picked it up? Or is it the final merge commit is what needs that prefix? Does every commit on a given branch of said category require a prefix with the commit message, or just the final when squashing? 

I wonder if shifting this to pulling off the branch name would be more ergonomic, since it's one and done regardless of the squash or collection of commit messages? Either way I'll make note of it if it's the merge commit. Remembered to squash since we did that in DISCO 👍 but indeed easy to forget.